Dockerfile

```diff
@@ -19,6 +19,7 @@ FROM gcr.io/distroless/base:latest
 WORKDIR /
 COPY --from=builder /workspace/baremetal-operator .
 USER nonroot:nonroot
+ENTRYPOINT = ["/baremetal-operator"]
```

Comment:
I don't think the `=` is [valid](https://docs.docker.com/engine/reference/builder/#entrypoint) here.

Reply:
Fixed